quic/s2n-quic-core/src/inet/ethernet.rs

```diff
@@ -0,0 +1,189 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::inet::Unspecified;
+use core::fmt;
+
+const MAC_LEN: usize = 48 / 8;
```

Comment:
Why "48/8" and not 6? Is the Mac address defined in terms of bits?

Reply:
Most of this networking stuff is defined in bits, yeah. I figured it would be a bit easier to tie documentation to the implementation here.